src/components/Dialog/CollectionsDialog.js

```diff
@@ -18,6 +20,8 @@ export const COLLECTIONS_DIALOG_ACTIONS = {
   DUPLICATE: 'DUPLICATE',
   DELETE: 'DELETE',
   DELETE_ERROR: 'DELETE_ERROR',
+  DISCARD: 'DISCARD',
```

Comment:
Since you have `DISCARD_CREATE`, maybe this can be `DISCARD_EDIT` to be more clear.